<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogcadastrarexercicios</class>
 <widget class="QDialog" name="Dialogcadastrarexercicios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#47cbd9</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="1" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #fff</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <item>
       <widget class="QLabel" name="nome_do_treino">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#20ab7d
</string>
        </property>
        <property name="text">
         <string>Nome do treino:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="texto_do_nome_do_treino">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">background-color:#fff</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="repeticao_1">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#20ab7d
</string>
        </property>
        <property name="text">
         <string>Repetição:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="exercicio_1">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#20ab7d
</string>
        </property>
        <property name="text">
         <string>Exercício:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="texto_rep_1">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="texto_exercicio_1">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="repeticao_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#20ab7d
</string>
        </property>
        <property name="text">
         <string>Repetição:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="exercicio_2">
        <property name="styleSheet">
         <string notr="true">color:#20ab7d
</string>
        </property>
        <property name="text">
         <string>Exercício:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="texto_rep_2">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="texto_exercicio_2"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="repeticao_3">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#20ab7d
</string>
        </property>
        <property name="text">
         <string>Repetição:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="exercicio_3">
        <property name="styleSheet">
         <string notr="true">color:#20ab7d
</string>
        </property>
        <property name="text">
         <string>Exercício:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="texto_rep_3">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="texto_exercicio_3"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="repeticao_4">
        <property name="styleSheet">
         <string notr="true">color:#20ab7d
</string>
        </property>
        <property name="text">
         <string>Repetição:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="exercicio_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#20ab7d
</string>
        </property>
        <property name="text">
         <string>Exercício:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="texto_rep_4">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="texto_exercicio_4"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="repeticao_5">
        <property name="styleSheet">
         <string notr="true">color:#20ab7d
</string>
        </property>
        <property name="text">
         <string>Repetição:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="exercicio_5">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#20ab7d
</string>
        </property>
        <property name="text">
         <string>Exercício:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLineEdit" name="texto_rep_5">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="texto_exercicio_5"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="repeticao_6">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#20ab7d
</string>
        </property>
        <property name="text">
         <string>Repetição:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLabel" name="exercicio_6">
        <property name="styleSheet">
         <string notr="true">color:#20ab7d
</string>
        </property>
        <property name="text">
         <string>Exercício:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLineEdit" name="texto_rep_6">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="texto_exercicio_6"/>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">background-color: #4CAF50;
border: none;
color: white;
padding: 15px 32px;
text-align: center;
text-decoration: none;
font-size: 16px;
margin: 4px 2px;
border-radius: 5px;</string>
        </property>
        <property name="text">
         <string>Cadastrar treino</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
